python/semantic_kernel/connectors/ai/chat_completion_client_base.py

```diff
@@ -59,7 +59,7 @@ def _prepare_chat_history_for_request(
         chat_history: "ChatHistory",
         role_key: str = "role",
         content_key: str = "content",
-    ) -> list[dict[str, str | None]]:
+    ) -> Any:
```

Comment:
Do we have a better typing for this?

Reply:
This function will potentially get overridden by other connectors, which would need more specific types.